lib/blockscore/actions/update.rb

```diff
@@ -0,0 +1,38 @@
+module BlockScore
+  module Actions
+    module Update     
+      def save
+        begin
+          self.class.patch "#{self.class.endpoint}#{id}", filter_params
+          true
+        rescue
+          false
+        end
+      end
+
+      # Filters out the non-updateable params
+      def filter_params
+        persistent = %i(id object created_at updated_at livemode class)
+        @attrs.reject { |k,v| persistent.include?(k) }
+      end
+
+      def method_missing(method, *args, &block)
+        if respond_to_missing? method
+          is_setter = method.to_s[-1] == '='
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.